google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/FormatOptions.java

```diff
@@ -120,6 +120,8 @@ public static FormatOptions of(String format) {
     checkArgument(!isNullOrEmpty(format), "Provided format is null or empty");
     if (format.equals(CSV)) {
       return csv();
+    } else if (format.equals(AVRO)) {
+      return avro();
```

Comment:
don't you need to also update the avro() function to use your new AvroOptions type?

Reply:
Yes, good catch. Done. Thanks!